src/main/java/org/team199/robot2022/subsystems/Climber.java

```diff
@@ -61,138 +72,68 @@ public Climber() {
-    }
-
-    public void extendLeft() {
-        left.set(kExtendSpeed);
-        SmartDashboard.putString("Left climber is", "Extending");
-    }
 
-    public void extendRight() {
-        right.set(kExtendSpeed);
-        SmartDashboard.putString("Right climber is", "Extending");
-    }
-
-    public void retractLeft() {
-        left.set(kRetractSpeed);
-        SmartDashboard.putString("Left climber is", "Retracting");
-    }
-
-    public void retractRight() {
-        right.set(kRetractSpeed);
-        SmartDashboard.putString("Right climber is", "Retracting");
-    }
+        dEncoderPos.put(EncoderPos.extendRight, 5.315);
+        dEncoderPos.put(EncoderPos.retractLeft, -1.151);
+        dEncoderPos.put(EncoderPos.extendLeft, -5.317);
+        dEncoderPos.put(EncoderPos.retractRight, -1.172);
+        dEncoderPos.put(EncoderPos.zero, 0.0);
 
-    public void slowExtendLeft() {
-        left.set(kSlowExtendSpeed);
-        SmartDashboard.putString("Left climber is", "Extending");
+        dMotorSpeed.put(MotorSpeed.retract, -((kDesiredRetractSpeedInps / kInPerSec) + (kVoltsToCounterTorque / 12))); // ~ -0.06151
+        dMotorSpeed.put(MotorSpeed.extend, (kDesiredExtendSpeedInps / kInPerSec)); // ~0.30261
+        dMotorSpeed.put(MotorSpeed.slowRetract, -((kSlowDesiredRetractSpeedInps / kInPerSec) + (kSlowVoltsToCounterTorque / 12))); // ~ -0.06151
+        dMotorSpeed.put(MotorSpeed.slowExtend, (kSlowDesiredExtendSpeedInps / kInPerSec)); // ~0.30261
     }
 
-    public void slowExtendRight() {
-        right.set(kSlowExtendSpeed);
-        SmartDashboard.putString("Right climber is", "Extending");
-    }
-    public void slowRetractLeft() {
-        left.set(kSlowRetractSpeed);
-        SmartDashboard.putString("Left climber is", "Retracting");
+    @Override
+    public void periodic() {
+        SmartDashboard.putNumber("L Climber Pos", leftEncoder.getPosition());
+        SmartDashboard.putNumber("R Climber Pos", rightEncoder.getPosition());
     }
 
-    public void slowRetractRight() {
-        right.set(kSlowRetractSpeed);
-        SmartDashboard.putString("Right climber is", "Retracting");
+    //motor = -1 left or 1 right or 0 both
+    public void resetEncodersTo(EncoderPos pos,int motor){
+			setEncoder[motor+1].accept(dEncoderPos.get(pos));
     }
 
-    public void stop() {
-        left.set(0);
-        right.set(0);
-        SmartDashboard.putString("Left climber is", "Stopped");
-        SmartDashboard.putString("Right climber is", "Stopped");
+    public void moveMotors(MotorSpeed speed, int motor){
+      setMotor[motor+1].accept(dMotorSpeed.get(speed));
     }
 
-    public void stopLeft() {
+    public void stopMotors(int motor){
+      if (motor<1){
         left.set(0);
         SmartDashboard.putString("Left climber is", "Stopped");
-    }
-
-    public void stopRight() {
+      }
+      if (motor>-1){
         right.set(0);
         SmartDashboard.putString("Right climber is", "Stopped");
```

Comment:
```suggestion
        SmartDashboard.putString("Right Climber State", "Stop");
```
\- keep the SmartDashboard keys/values consistent to the rest of the code